CHANGELOG.md

```diff
@@ -111,5 +111,6 @@ The main features of this release are:
 - [\#184](https://github.com/arkworks-rs/algebra/pull/184) Compile with `panic='abort'` in release mode, for safety of the library across FFI boundaries.
 - [\#192](https://github.com/arkworks-rs/algebra/pull/192) Fix a bug in the assembly backend for finite field arithmetic.
 - [\#217](https://github.com/arkworks-rs/algebra/pull/217) (ark-ec) Fix the definition of `PairingFriendlyCycle` introduced in #190.
+- [\#252](https://github.com/arkworks-rs/algebra/pull/252) (ark-ff) Fix prime field sampling when `REPR_SHIFT_BITS` is 64.
```

Comment:
Let me make a PR to update the changelog on master, this should go in a new release's section

Reply:
Oh right! Please do so.

Reply:
I did this in  https://github.com/arkworks-rs/algebra/pull/230

Reply:
Got it. It turns out that I just did not notice that there is already a pending section above.